PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -96,6 +96,8 @@
                 windowViews.setObject(snapshotStatus, forKey: view)
             }
 
+            // TODO: IncrementalSnapshot, type=2
```

Comment:
probably not needed if we make the `screenshot` the default option since it is always a single image anyway, unless at some point we try to optimize the data transfer by breaking a full image in tiles and only sending the ones that have changed.